framework_tvb/tvb/core/adapters/abcadapter.py

```diff
@@ -463,17 +464,17 @@ def load_entity_by_gid(data_gid):
 
     @staticmethod
     def load_traited_by_gid(data_gid):
-        # type: (uuid.UUID) -> HasTraits
+        # type: (typing.Union[uuid.UUID, str]) -> HasTraits
         """
         Load a generic HasTraits instance, specified by GID.
         """
-        index = load_entity_by_gid(data_gid.hex)
+        index = ABCAdapter.load_entity_by_gid(data_gid.hex)
```

Comment:
.hex should be removed here

Reply:
Changing this line (including Paula's suggestion) actually fixed one unit-test, without us actively expecting that 👍